src/view/home.js

```diff
@@ -41,21 +41,26 @@ const HomeView = ({
   transaction,
   nav,
 }) => {
-  const { depositLabel, channelBalanceLabel, unitLabel } = store;
+  const {
+    totalBalanceLabel,
+    unitLabel,
+    channelStatus,
+    channelPercentageLabel,
+  } = store;
   return (
     <Background image="purple-gradient-bg">
       <HomeHeader
-        goChannels={() => channel.init()}
+        goDeposit={() => nav.goDeposit()}
```

Comment:
I got pretty confused when trying to find where the channels page went. I don't find clicking `95% on Lightning` very discoverable. 

One vote for the header going back to how it was before, or at least separating this feature into a different PR for further discussion :)

Reply:
> I don't find clicking `95% on Lightning` very discoverable.

I think we agreed to test this and get feedback from users as it's released. We can always iterate and improve on this later.